schemas/stsci.edu/asdf/unit/quantity-1.1.0.yaml

```diff
@@ -21,6 +18,7 @@ properties:
         items:
           type: number
```

Comment:
I believe it's the case that nested lists will be handled by `ndarray` since this is the type that `astropy.Quantity` uses to represent non-scalar values anyway. I think I would argue that if you include a literal nested array in a manually edited ASDF file, it should really be an `ndarray` anyway. Does that make sense? I don't think ASDF currently provides support for arbitrarily nested arrays outside of `ndarray`.